docs/manual/docs/cmd/pa/connector/connector-list.md

```diff
@@ -0,0 +1,38 @@
+# pa connector list
+
+Lists custom connectors in the given environment
+
+## Usage
+
+```sh
+pa connector list [options]
```

Comment:
`pa` refers to `PowerApps` or `PowerAutomate` 🔮 🤣? Microsoft never makes it easy, aren't they?